doc/Comments.xml

```diff
@@ -150,17 +151,31 @@ Example:
 #!  This is my chapter.
 #!  I document my stuff in it.
 ]]></Listing>
+
+Labels of chapters (for referencing purposes) are <C>Chapter_</C><A>name</A>, where
+all spaces in <A>name</A> are replaced by underscores. The <C>@ChapterLabel</C> <A>label</A> command
+can be used to set a custom label. This will set the label of the chapter to <C>Chapter_</C><A>label</A>.
```

Comment:
"This will" is unclear -- what is "this"? Perhaps write this:

"You can set a custom label by using the <C>@ChapterLabel</C><A>label</A> command, which will set the label of the chapter to <C>Chapter_</C><A>label</A>."